### ISSUE TITLE
Flaky-test: MultiTopicsConsumerTest.testShouldMaintainOrderForIndividualTopicInMultiTopicsConsumer

### ISSUE THREAD

**Issue by shibd** [apache/pulsar#14558]
MultiTopicsConsumerTest.testShouldMaintainOrderForIndividualTopicInMultiTopicsConsumer is flaky. It fails sporadically.

[example failure](https://github.com/apache/pulsar/runs/5418948651?check_suite_focus=true#step:9:1741)

```
Error:  Tests run: 5, Failures: 1, Errors: 0, Skipped: 3, Time elapsed: 312.09 s <<< FAILURE! - in org.apache.pulsar.client.api.MultiTopicsConsumerTest
Error:  testShouldMaintainOrderForIndividualTopicInMultiTopicsConsumer(org.apache.pulsar.client.api.MultiTopicsConsumerTest)  Time elapsed: 241.064 s  <<< FAILURE!
java.util.concurrent.TimeoutException
	at java.base/java.util.concurrent.CompletableFuture.timedGet(CompletableFuture.java:1886)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2021)
	at org.apache.pulsar.client.api.MultiTopicsConsumerTest.testShouldMaintainOrderForIndividualTopicInMultiTopicsConsumer(MultiTopicsConsumerTest.java:188)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
	at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:45)
	at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:73)
	at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

**Comment by shibd** [comment]
related  #12456

**Comment by lhotari** [comment]
fixed by #14572
